src/Compilers/CSharp/Portable/Symbols/DelegateCaches/ModuleScopedDelegateCacheContainerField.cs

```diff
@@ -0,0 +1,40 @@
+﻿// Copyright (c) Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information.
+
+using System;
+using System.Collections.Generic;
+using System.Collections.Immutable;
+using System.Diagnostics;
+using System.Runtime.InteropServices;
+using Microsoft.Cci;
+using Microsoft.CodeAnalysis.CodeGen;
+using Microsoft.CodeAnalysis.CSharp.Syntax;
+using Roslyn.Utilities;
+
+namespace Microsoft.CodeAnalysis.CSharp.Symbols
+{
+    internal class ModuleScopedDelegateCacheContainerField : SynthesizedFieldSymbol
+    {
+        /// See comments on <see cref="ModuleScopedDelegateCacheContainer.SortKey"/>
```

Comment:
It doesn't look like ModuleScopedDelegateCacheContainer.SortKey has any doc comment. Also, it is not clear how that comment would be relevant for the _name.